documentation/common/address-space.adoc

```diff
@@ -127,3 +127,52 @@ The multicast address type is a scalable direct address for sending messages to
 * standard
 
 // address.multicast.plan.inmemory.shortDescription:Configures router network with multicast address
+
+[[brokered_address_space]]
+=== Brokered Address Space
+
+The brokered address space is designed to support broker-specific features, at the cost of limited
+scale in terms of the number of connections and addresses. This address space supports JMS
+transactions, message groups etc. The brokered address space is backed by a broker, and stores
+persisted messages on disk if required.
```

Comment:
The last sentence seems a little redundant to me. Its not wrong though in any way. I think whats there is good to start off with and we can then try and iterate on it as time allows.